src/protocols/ethereum/helpers/index.ts

```diff
@@ -0,0 +1,26 @@
+import { isAddress } from 'web3-validator';
+import { Web3Eth } from 'web3-eth';
+import { fromWei, toWei } from 'web3-utils';
+import BigNumber from 'bignumber.js';
+
+export function isEthAddressValid(address: string) {
+  return isAddress(address);
+}
+
+/**
+ * Return the base fee of the latest block in Ether
+ */
+export async function getEthBaseFee() {
+  // TODO: use our own node
+  const web3Eth = new Web3Eth('https://eth.llamarpc.com');
```

Comment:
The node URL should be set in the network settings in my opinion.

Reply:
Yes, once the adapter is completed we can change this dummy code

Reply:
Yeah, but then the architecture and the source of this url will be different and I'm afraid that having just an utility function would not be possible - we would need to refactor it havilly. This is why I'm suggesting to mock this url in the place where it will be stored in the future to have the architecture done in a proper way now.